2019-01/PITCHME.md

```diff
@@ -176,5 +176,32 @@ Note:
 Still for the context, other projects we are using behind the scenes.
 
 ---
-@title[What's in it for SUSE?]
+@title[What about SUSE in the community?]
 
+#### What about SUSE in the community?
+
+@ul[](false)
+- OpenStack tests against openSUSE Leap using our packaging
+- When it breaks, they sometimes ping dirk on irc
+- And then... Sometimes... dirk is not available!
+- We need a better, more sustainable support for the community and take care of issues before they arise
+@ulend
+
+Help support OpenSUSE in OpenStack by:
+
+@ul[](false)
+- Helping provide stable support for OpenSUSE images by helping maintain diskimage-builder, supporting the mirrors for the OpenDev (nee OpenStack) Infra team
+- Help support community packages by contributing to the rpm-packaging project and keeping the Cloud:OpenStack:* OBS projects stable
```

Comment:
its not only about Cloud:OpenStack: . but about all of the distro. e.g. we released an unbound(1) security update that broke it completely (and that broke the CI)